### PR TITLE
Add digital marketplace frontend toolkit

### PR DESCRIPTION
**Pull request by tombye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,3 +104,16 @@ node_modules/gulp/bin/gulp.js build:development
...
+The Digital Marketplace front-end toolkit is included for use in all front-end coding.
+
+#### Updating
+
+To update the Digital Marketplace front-end toolkit, run the `scripts/update-dm-toolkit.sh` script, passing in the version you want to update it to.
+
+For example, to update it to version 0.12.0, run the following:
+
+```
+./scripts/update-dm-toolkit.sh 0.13.0
```
Comment (TheDoubleK) [inline README.md]: It seems counterintuitive to specify '0.13.0' in order to install '0.12.0' ;)

**Comment by TheDoubleK** [comment]
This PR does not seem to include a Favicon.

**Comment by TheDoubleK** [comment]
As the toolkit is a separate project do we want to have it's source under version control here too? Other options would be to either (1) add it as a submodule, or (2) git ignore the ```digitalmarketplace-frontend-toolkit``` folder and just have the 'update toolkit version' script as part of this project, which gets run as part of the bootstrapping process.

What do you think?  I'm happy to have it here if that's best, but doesn't seem quite right to have the toolkit sources versioned here if really all we should do is pull in different release versions of the toolkit.

**Comment by tombye** [comment]
Raised an issue for the dependencies problem: https://github.com/TheDoubleK/digitalmarketplace-thermos/issues/10

**Pull request merged**